### PR TITLE
Fix publisher and author name

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "displayName": "atlas",
   "description": "Connects vscode with different server adapters",
   "version": "0.0.2",
-  "publisher": "mohitsuman",
+  "publisher": "Red Hat Developers",
```
Comment (mohitsuman) [inline package.json]: The publisher name is incorrect here. It should be this: ``` "publisher": "redhat",```

Comment (mohitsuman) [reply package.json]: And also add and author field mentioning the following:
```"author": "Red Hat",```

Comment (sudhirverma) [reply package.json]: done

File: package.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "displayName": "atlas",
   "description": "Connects vscode with different server adapters",
   "version": "0.0.2",
-  "publisher": "Red Hat Developers",
+  "publisher": "RedHat",
```
Comment (mohitsuman) [inline package.json]: @sudhirverma It should be **redhat** in smallcase, as mentioned earlier.

**Pull request merged**